.github/workflows/arduino.yml

```diff
@@ -0,0 +1,170 @@
+name: Arduino CI Build
+
+# See https://github.com/wolfSSL/Arduino-wolfSSL
+
+# TODO remove '*' om push/ branches, and uncomment github.repository_owner == 'wolfssl'
```

Comment:
Is it possible to clean this TODO up now or does it have to be a future PR? Maybe it can be cleaned up after https://github.com/wolfSSL/wolfssl/pull/8514 is merged?

Reply:
That particular TODO for the Arduino workflow can be changed now (done), but  others in the `run` section must have https://github.com/wolfSSL/wolfssl/pull/8514 merged to run without error.

It should be noted that the currently published Arduino library v5.7.6 will also fail for those examples for the same reason.